openpype/modules/ftrack/event_handlers_user/action_create_folders.py

```diff
@@ -51,13 +55,33 @@ def interface(self, session, entities, event):
             },
             {
                 "type": "label",
-                "value": "With all chilren entities"
+                "value": "Create with all children entities"
             },
             {
                 "name": "children_included",
                 "type": "boolean",
                 "value": False
             },
+            {
```

Comment:
This is misleading label. Something like "Create folders only for entities with tasks" would be more suitable.

Reply:
I can't quite tell exactly what this does either actually

Reply:
Well the children included is somethintg that was already there, i don't know why you would need the check, maybe it was some feature request of a studio?

Reply:
Aside of the fact that if you uncheck the children included the script actually does nothing.

Reply:
The question is about `Create publish and work folders`

`Children included` should generate folder for all children as well as selected entity. so If you generate for `sequence`. With the boolean on, you will get all the shots as well, with it Off, you will only get the sequence.

Reply:
If the intention is that it will skip creating publish and work folders for entities that are other than `shots` and `asset builds` than that is way too specific to your workflow. We have users that have entities named `poster`, `book`, `library` and so on and they want these folders generated there. We can't make such assumption